internal/credential/service_store.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Can this file be combined into `service_list_stores.go`? For libraries and credentials that seems to be the pattern. This is the only one that has a separate `service_<type>.go` file.

Reply:
I will merge it, but the reason it was separate was because it is now the only resource which has a service struct at this level. Prior to the libraries refactor it too had a `service_library.go` file.